utils/das_data_sorter.py

```diff
@@ -1,39 +1,43 @@
 import pandas as pd 
 from argparse import ArgumentParser
-from numpy import ceil
+from numpy import ceil, median
 
 # accepts terminal arguments
 parser = ArgumentParser()
-parser.add_argument("--input", help="Reads the inputted CSV file to filter", action="store", required=True)
-parser.add_argument("--output", help="Writes the filtered data onto a new CSV file under this name", action="store", required=True)
-parser.add_argument("--unit", help="Specify time units [seconds, s, or minutes, m]. Default is in seconds.", default="seconds", 
+parser.add_argument("-i", "--input", help="Reads the inputted CSV file to filter", action="store", required=True)
+parser.add_argument("-o", "--output", help="Writes the filtered data onto a new CSV file under this name", action="store", required=True)
+parser.add_argument("--unit", help="Specifies time units [seconds, s, or minutes, m]. Default is in seconds.", default="seconds", 
```

Comment:
I know this was added in a different PR, but you don't need to specify what units are valid `[seconds, s, or minutes, m]` because the `--help` will do that for you.
![Screen Shot 2020-06-20 at 12 06 53 am](https://user-images.githubusercontent.com/18709969/85141326-f3df9000-b289-11ea-9244-0ba09dd14865.png)